tests/test_ecs.py

```diff
@@ -150,6 +150,66 @@ def test_ecs_service_task_def_filter(self):
         self.assertEqual(len(resources), 1)
         self.assertEqual(resources[0]["serviceName"], "home-web")
 
+    def test_ecs_service_tag(self):
+        session_factory = self.replay_flight_data(
```

Comment:
please switch the tag/remove-tag test to task-definition, it needs much less recorded json since its not a hierarchical resource, and its the same underlying implementation thats supporting both. its also possible to combine the remove-tag/tag into a single test like in test_ecr test_ecr_tags. at the moment your recording has tons of extraneous clusters and services which is causing at least a 1k of extra recording here. there's a bunch of re-recording in this pr on existing tests for reasons that aren't entirely clear as well, you can copy in a list tags response into existing tests.

Reply:
switched the tagging tests from service to task-definition and condensed the tag/remove-tag tests into one, resulting in a large drop in the amount of json recorded